WeakAuras/RegionTypes/AuraBar.lua

```diff
@@ -381,6 +380,14 @@ local barPrototype = {
             extraTexture:SetVertexColor(1, 1, 1, 1);
           end
 
+          local texture = self.additionalBarsTextures and self.additionalBarsTextures[index];
+          if texture then
+            local texturePath = SharedMedia:Fetch("statusbar", texture) or ""
+            extraTexture:SetTexture(texturePath, extraTextureWrapMode, extraTextureWrapMode)
```

Comment:
I did two minor changes here:
a) I added the extraTextureWrapMode, which was missing and
b) I added the else branch for setting the texture, which is required if e.g. a aurabar is reused with different extra texture settings.. (Note that previously that was also buggy.)